squidlib-util/src/test/java/squidpony/squidmath/NumberToolsTest.java

```diff
@@ -0,0 +1,109 @@
+package squidpony.squidmath;
+
+import org.junit.Assert;
+import org.junit.Test;
+import squidpony.squidmath.NumberTools;
+
+public class NumberToolsTest {
+
+  @Test
+  public void atan2_Test() {
```

Comment:
Would recommend `testAtan2`, `testCos`, `testSway` style names in keeping with the existing tests.

Reply:
Resolved